testkit/src/test/java/org/apache/calcite/test/DiffRepositoryTest.java

```diff
@@ -44,4 +44,17 @@ public class DiffRepositoryTest {
     }
     assertThat("First assertion must always fail", assertPassed, is(false));
   }
+
+  @Test void testMethodOnlyExistsInXml() {
+    boolean assertPassed = false;
+    DiffRepository r = DiffRepository.lookup(DiffRepositoryTest.class);
+    final String actual = "testMethodOnlyExistsInXml1";
```

Comment:
There's a small question, should this be "testMethodOnlyExistsInXml1_ actual"?

Reply:
You can refer to the comments above about the detailed exception info, both are acceptable.

Reply:
The file here should be `DiffRepositoryTest_actual.xml`.

Reply:
I understand, this may just be a placeholder.